rotkehlchen/tests/api/test_custom_assets.py

```diff
@@ -824,3 +830,101 @@ def assert_db() -> None:
+    """Test that the endpoint for exporting custom assets works correctly"""
+    eth_address = make_ethereum_address()
+    identifier = f'_ceth_{eth_address}'
+    globaldb.add_asset(
+        asset_id=identifier,
+        asset_type=AssetType.ETHEREUM_TOKEN,
+        data=EthereumToken.initialize(
+            address=eth_address,
+            decimals=18,
+            name='yabirtoken',
+            symbol='YAB',
+            coingecko='YAB',
+            cryptocompare='YAB',
+        ),
+    )
+    with tempfile.TemporaryDirectory() as path:
+        if with_custom_path:
+            response = requests.put(
+                api_url_for(
+                    rotkehlchen_api_server,
+                    'userassetsresource',
+                ), json={'action': 'download', 'destination': path},
+            )
+        else:
+            response = requests.put(
+                api_url_for(
+                    rotkehlchen_api_server,
+                    'userassetsresource',
+                ), json={'action': 'download'},
+            )
+
+        result = assert_proper_response_with_result(response)
+        if with_custom_path:
+            assert path in result['file']
+        zip_file = ZipFile(result['file'])
+        data = json.loads(zip_file.read('assets.json'))
+        assert int(data['version']) == GLOBAL_DB_VERSION
+        assert len(data['assets']) == 1
+        assert data['assets'][0] == {
+            'identifier': identifier,
+            'name': 'yabirtoken',
+            'decimals': 18,
+            'symbol': 'YAB',
+            'asset_type': 'C',
+            'started': None,
+            'forked': None,
+            'swapped_for': None,
+            'cryptocompare': 'YAB',
+            'coingecko': 'YAB',
+            'protocol': None,
+            'underlying_tokens': None,
+            'ethereum_address': eth_address,
+        }
+        assets_file = Path(result['file'])
+        assets_directory = assets_file.parent
+        assets_file.unlink()
+        assets_directory.rmdir()
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('start_with_logged_in_user', [True])
+@pytest.mark.parametrize('method', ['post', 'put'])
+def test_importing_custom_assets_list(rotkehlchen_api_server, method):
+    """Test that the endpoint for importing custom assets works correctly"""
+    dir_path = Path(__file__).resolve().parent.parent
+    filepath = dir_path / 'data' / 'exported_assets.json'
+
+    if method == 'put':
+        response = requests.put(
+            api_url_for(
+                rotkehlchen_api_server,
+                'userassetsresource',
+            ), json={'action': 'upload', 'file': str(filepath)},
+        )
+    else:
+        response = requests.post(
+            api_url_for(
+                rotkehlchen_api_server,
+                'userassetsresource',
+            ), json={'action': 'upload'},
+            files={'file': open(filepath, 'rb')},
+        )
```

Comment:
you should also chekc the response is fine here. With `assert_simple_ok_response` or whatever function is needed depending on response type